site/examples/tooltips.js

```diff
@@ -6,6 +6,27 @@ const basic = `<div class='flex-parent-inline flex-parent--center-main flex-pare
   <span class='triangle triangle--d'></span>
 </div>`;
 
+const basic_left = `<div class='flex-parent-inline flex-parent--center-cross'>
+  <span class='triangle triangle--l'></span>
+  <div class='bg-darken75 color-white align-center col col--12 p6 round txt-bold txt-s'>Hello world!</div>
```

Comment:
A couple of flexbox points:

- The flex children should have `flex-child` classes.
- We shouldn't need to use `col col--12` on these divs. If the desired effect is to have the triangle take up the space it needs and the body take up the rest of the space, we can do that like so:

  ```html
  <div class='flex-child flex-child--no-shrink'><!-- triangle in here --></div>
  <div class='flex-child flex-child--grow'><!-- body in here --></div>
  ```

  If the parent, though, is `flex-parent-inline`, I'm not sure you need to include the `--no-shrink` and `--grow` modifiers are all?